.docker/drupal/setup.sh

```diff
@@ -34,6 +50,5 @@ echo "##########################################################################
 ##############################################################################################"
 
 # Create a file to indicate installation has finished
-touch /root/.drupal-installation-complete
-
-apachectl start -DFOREGROUND
+touch ~/.drupal-installation-complete
```

Comment:
I'm wondering, can we use the pidfile from the upservisord config for this instead?

Reply:
happy to look at that in a follow up

Reply:
Sure! Create a "easy-pickup" task :)